### ISSUE TITLE
MM Submission Tab - Improvements 

### ISSUE THREAD

**Issue by dushyantb** [topcoder-platform/community-app#3374]
1. member handle color coding
2. allow admin to view the additional details even though he is not  registered to the challenge
3. in the console log, if I’m not registered, it throws a 500 and a raw text response of Cannot read property 'userDetails' of undefined - instead of throwing a 401 and a json payload
4. Speed-up loading submissions

**Comment by ThomasKranitsas** [comment]
@dushyantb req.2 should be already implemented:
https://github.com/topcoder-platform/community-app/blob/65dab4abaa86ef9b8eb80ab4ebd0cd9a8ffac747/src/server/controllers/submissions.js#L91-L94

Regarding the error on req.3, where exactly is the error thrown?

**Comment by dushyantb** [comment]
@ThomasKranitsas nope, req.2 is not working though code suggests that it is working.
- Regarding req.3 I'll confirm
